src/mousedraghandler.cpp

```diff
@@ -0,0 +1,173 @@
+#include "client.h"
+#include "monitormanager.h"
+#include "mouse.h"
+#include "mousedraghandler.h"
+#include "tag.h"
+#include "x11-utils.h"
+
+MouseDragHandler::MouseDragHandler(ClientManager*, MonitorManager*  monitors, Client* dragClient, MouseDragFunction function)
```

Comment:
Did you leave out the parameter name here on purpose? It seems a bit inconsistent (cf. MonitorManager).

There's also some double whitespace (in the header file as well)

Reply:
I don't need `ClientManager*` here, but I thought that I might need it at some point.. Since it's not much work, I've removed the unused `ClientManager*` from the constructor.